kolena/_utils/instrumentation.py

```diff
@@ -128,11 +128,14 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
                 event_metadata["response_error"] = e.__class__.__name__
                 raise e
             finally:
-                event_metadata["duration"] = round((datetime.datetime.now() - start_time).total_seconds(), 3)
+                event_metadata["duration"] = round(
```

Comment:
The 2 ignores can be resolved by specifying typing for `event_metadata` above:

```
event_metadata: Dict[str, Union[str, int, float, bool, None]] = {}
```

Reply:
Done